docs/source/testing/landing-page.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Setting an Apollo Server landing page
+sidebar_title: Server landing page
```

Comment:
Not sure this is the most helpful sidebar title, esp under "testing". Like this page documents the landing page, yes, but the reason it's under testing is because it gives you access to an interface that lets you run operations against your server. Not sure I have a better suggestion though.